### DISCUSSION TITLE
Install Pi VPN alongside Open Media Vault

### DISCUSSION THREAD

**Discussion by d4rklynk** [pivpn/pivpn#1664] [Q&A]
Hi, I was wondering if I can use Pi VPN alongside an Open Media Vault installation without breaking anything.

**Comment by coolapso** [comment]
https://docs.pivpn.io/faq/#pivpn-supports-the-following-systems


Anything else you're on your own risk.
OMV is debian based, in theory it should work but I don't know what changes does OMV has in the system to guarantee that nothing will break.
